opentracing-spring-web-handler-interceptor/src/main/java/io/opentracing/contrib/spring/web/servlet/TracingHandlerInterceptor.java

```diff
@@ -0,0 +1,102 @@
+package io.opentracing.contrib.spring.web.servlet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.contrib.web.servlet.filter.HttpServletRequestExtractAdapter;
+import io.opentracing.contrib.web.servlet.filter.TracingFilter;
+import io.opentracing.propagation.Format;
+import io.opentracing.tag.Tags;
+
+/**
+ * Tracing handler interceptor for spring web. It creates a new span for an incoming request.
+ * Created span can be accessed via {@link HttpServletRequest#getAttribute(String)} with name
+ * {@link TracingHandlerInterceptor#ACTIVE_SPAN}.
```

Comment:
This seems like it would severely reduce the usefulness of the instrumentation, since the span is not propagated to any outbound calls this service might make. Basically, the instrumentation like this is most useful when there's an established standard for in-process context/span propagation.